tests/jobs/scala/src/main/scala/KerberizedKafkaProducer.scala

```diff
@@ -0,0 +1,60 @@
+import java.util.{HashMap => JMap}
+
+import org.apache.kafka.clients.producer.{KafkaProducer, ProducerConfig, ProducerRecord}
+
+import scala.util.{Failure, Success, Try}
+
+/**
+  * A simple Kafka Producer which should be used in the same way as kafka-console-producer with Kerberized
+  * Kafka instance. The idea here is that producer runs as a driver without the need to launch executors.
+  */
+object KerberizedKafkaProducer {
+  def main(args: Array[String]): Unit = {
+    if (args.length < 4) {
+      throw new IllegalArgumentException("USAGE: <service name> <brokerlist> <topic> [message1, message2, ...]")
+    }
+
+    val Array(service, brokers, topic) = args.take(3)
+    val messages = args.drop(3)
+    println(s"Got brokers $brokers, producing ${messages.length} to topic $topic")
+    try {
+      withProducer(service, brokers, sendMessages(topic, messages))
+    } catch {
+      case t: Throwable =>
+        t.printStackTrace()
+        Thread.sleep(30*60*1000)
```

Comment:
why sleep for 30 minutes?

Reply:
This was added to ease Kafka producer troubleshooting in case of errors (happens only when an exception is thrown). After verifying functionality it looks like it's not needed anymore.